app/agents/voice/automatic/tts/__init__.py

```
@@ -57,6 +57,53 @@ def get_tts_service(
+    )
+
+    if config.GOOGLE_CREDENTIALS_JSON:
+        # Log first 200 characters to check format
+        credentials_preview = config.GOOGLE_CREDENTIALS_JSON[:200]
+        logger.info(
+            f"[TTS] Google credentials preview (first 200 chars): {credentials_preview}"
+        )
+
+        # Check for common JSON issues
+        if "\n" in config.GOOGLE_CREDENTIALS_JSON:
+            logger.warning(f"[TTS] Google credentials contains newline characters")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)